app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
```

Comment:
If you want to receive emails from devise for sign up, you need to take this :confirmable part and put it with the other plugins for devise (like after :validatable on the next line)